src/components/CreateProject/CreateProjectModal.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
```

Comment:
In our previous stories we used `Meta` and `Story`.  isn't better to use it like this: 
```suggestion
import type { Meta, Story } from '@storybook/react';
```

BTW, we should discuss about this in Refactoring.

Reply:
if you use `ComponentMeta` you'll have auto-completion in `argTypes` and the same for `ComponentStory`

Reply:
@AmirabbasJ what do you think?

Reply:
I checked it out 
IMO it's way better than using the Story and Meta
as you said it allows us to avoid exporting the props 
I agree with approach

Reply:
Good then.